common/src/main/java/com/wynntils/models/items/items/game/EmeraldItem.java

```diff
@@ -12,7 +12,8 @@ public class EmeraldItem extends GameItem implements EmeraldValuedItemProperty {
     private final Supplier<Integer> amountSupplier;
     private final EmeraldUnits unit;
 
-    public EmeraldItem(Supplier<Integer> amountSupplier, EmeraldUnits unit) {
+    public EmeraldItem(int emeraldPrice, Supplier<Integer> amountSupplier, EmeraldUnits unit) {
+        super(emeraldPrice);
```

Comment:
So this just looks very suspicious. In what way can the emeraldPrice be anything other than the amount of this item? And the latter is a supplier, but the former is fixed.

Reply:
I agree that the supplier is a problem. Items should not suddenly change their value.

RIght, the Shady merchant... :-) Ok, fair enough, I agree that emeralds can have a price that is not the face value.